templates/kubernetes/index.html

```diff
@@ -283,42 +284,55 @@ <h2 class="p-muted-heading u-align--center">FULLY-SUPPORTED KUBERNETES PRIVATE C
           }}
         </li>
         <li class="p-inline-images__item">
-          {{
-            image(
-            url="https://assets.ubuntu.com/v1/760109ef-CloudSigma_Logo_Full.png",
-            alt="Cloud Sigma",
-            width="130",
+          {{ image (
```

Comment:
I think [the inline images pattern](https://vanillaframework.io/docs/patterns/inline-images) is deprecated so we need to use [the logo section pattern ](https://vanillaframework.io/docs/patterns/logo-section)instead.